lib/slack-ruby-bot-server/config.rb

```diff
@@ -3,11 +3,24 @@ module Config
     extend self
 
     attr_accessor :server_class
+    attr_accessor :database
 
     def reset!
       self.server_class = SlackRubyBotServer::Server
     end
 
+    def database
+      database ||= File.file?('config/mongoid.yml') ? :mongo : :postgresql
```

Comment:
I think the right way to check is to look for `defined?(::Mongoid)` and such, also should fail if neither Mongoid or PostgreSQL are available.